src/main/java/BasePage.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+    protected WebDriver driver;
+
+    public BasePage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    // Enter text to text field
+    public static void enterTextInTextField(WebElement textField, String inputText) {
```

Comment:
Make it and all methods in this class `protected` if needed

Reply:
Done